From f61382eab72a99448c434de42f68822f091b32a0 Mon Sep 17 00:00:00 2001
From: Alan Tull <alan.tull@freescale.com>
Date: Tue, 13 Mar 2012 11:39:54 -0500
Subject: [PATCH 283/505] ENGR00174062 change clock enable_count to usecount

'enable_count' to chaned to 'usecount' in the clock code, so
change it in the dump-clocks.sh script to align.

Signed-off-by: Alan Tull <alan.tull@freescale.com>
---
 test/mxc_clock_test/dump-clocks.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_clock_test/dump-clocks.sh b/test/mxc_clock_test/dump-clocks.sh
index 9c42d8a..a57e6d2 100755
--- a/test/mxc_clock_test/dump-clocks.sh
+++ b/test/mxc_clock_test/dump-clocks.sh
@@ -15,7 +15,7 @@ for foo in $(find /sys/kernel/debug/clock -type d); do
 
     cd $foo
 
-    ec="$(cat enable_count)"
+    use="$(cat usecount)"
     rate="$(cat rate)"
 
     clk="$(basename $foo)"
@@ -26,7 +26,7 @@ for foo in $(find /sys/kernel/debug/clock -type d); do
         parent="   ---"
     fi
 
-    printf "%-24s %-24s %2d %10d\n" "$clk" "$parent" "$ec" "$rate"
+    printf "%-24s %-24s %2d %10d\n" "$clk" "$parent" "$use" "$rate"
 
     cd $saved_path
 done
-- 
1.8.0

